src/App.jsx

```diff
@@ -15,7 +16,7 @@ const App = () => {
       <Route path="*" element={<Home/>} />
       <Route path="/login" element={<Login saveUser={setUser}/>} />
       <Route path="/signup" element={<SignUp saveUser={setUser}/>} />
-        {user && <Route path="/wallet" element={<Wallet user={user}/>} />}
+        {user && <Route path="/wallet" element={ <> <Nav clearUser={setUser}/> <Wallet user={user}/> </>} /> }
```

Comment:
Could you format this better, so it's clear what's happening within the element?
E.g. 
![image](https://user-images.githubusercontent.com/97879313/200858153-24c891c9-7242-4353-9264-ecb89d0e61c1.png)